reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+manage production workloads by ensuring the core is stable and reliable.
+This means the features are:
+- Observable - you can tell that it is in use and working properly, and are
+  able to define reasonable service level objectives for the feature.
+- Supportable - the feature is well documented with a playbook covering failure
+  modes, dependencies and what happens when those fail or degrade, and a
+  troubleshooting guide.
+- Scalable - the feature does not introduce scaling issues.
+- Recoverable - the feature can be disabled or rolled back easily and without
+  data loss
+        
+### The Words We Use
+
+The Naming Working Group was spun up to undergo research and create decision
+making frameworks around the terminology we use in the technical components and
+the documentation.
+
+
+GitHub has also stepped up to help the open source community at large create a
+smooth path for `master->main` branch renaming. We discovered some gaps in our
+tooling and automation around this as well, but now have a [clear path]. A
+number of repos have already started this migration, and we will continue to 
+roll
+it out to the remainder of the org.
+   
+
+Not only is this a WG Naming initiative, but several parts of the project 
+reported in ways they are examining the words we use. For example, SIG 
+Contributor Experience's slack-infra team implemented a [new bot] for inclusive 
+language. SIG Testing has made a significant effort to eliminate [`blacklist`] 
+from the code base.
+   
+There is still plenty of work however in evaluating further language,
+implementing changes to code and documentation, as well as the testing and
+validation that related code changes don't introduce unexpected regressions.
+  
+[new bot]: https://sigs.k8s.io/slack-infra/slack-moderator-words
+[clear path]: https://www.kubernetes.dev/resources/rename/
+[`blacklist`]: https://github.com/kubernetes/community/pull/5341
+
+
+### All eyes on SECURITY.md
+
+This year brought the creation of [SIG Security], in response to the greater
+community and industry focus on the security of critical pieces of software
+like Kubernetes. This new SIG grew out of the previous Security Audit Working
+Group, and is designed to be a clear home for security-focused discussions
+across the project.
+
+
+This new group, partnering with the existing Product Security Committee, will
+focus on horizontal security initiatives for the Kubernetes project, including
+regular security audits, the vulnerability management process, cross-cutting
+security documentation, and security community management.
+
+
+The Product Security Committee is also currently discussing a
+[name change to "Security Response Committee"] to better reflect the role they
+play in security response.
+
+
+[SIG Security]: https://git.k8s.io/community/sig-security/charter.md
+[name change to "Security Response Committee"]: https://github.com/kubernetes/community/pull/5597
+
+
+### Did someone say KEPs?
```

Comment:
```suggestion
### Improving Kubernetes Enhancements
```

It's cute and I like the original, but someone who isn't us won't get it and won't understand this section without a better title.